src/components/NavButton/index.jsx

```diff
@@ -35,7 +41,7 @@ const GordonNavButton = ({
       </ListItem>
     ) : (
       <NavLink end to={linkPath} onClick={onLinkClick} className="gc360_link">
-        <ListItem divider={divider} button>
+        <ListItem divider={divider} button className="gc360_link">
```

Comment:
This added class is redundant with the class on the `NavLink`. You should only need one of these.

Reply:
So, the link buttons were still displaying as different colors again when I only had them on the NavLink component, I can remove them from the navLink component, which seems to work from testing, so I'll commit that for review.